frontend/src/components/common/Resource/EditorDialog.tsx

```diff
@@ -273,6 +321,36 @@ export default function EditorDialog(props: EditorDialogProps) {
       setError(t("Error parsing the code. Please verify it's valid YAML or JSON!"));
       return;
     }
+
+    const newItemDefs = obj!;
+
+    if (!applyOnSave) {
```

Comment:
This is essentially checking whether or not to use an external dispatch (formerly called `externalDispatch`). The name might be a bit confusing so I think it could be changed to something closer to what it was originally, but setting this to true means no dispatch is performed when creating a resource and 2 dispatches are performed when editing